addons/html_builder/static/src/website_preview/website_builder_action.js

```diff
@@ -133,14 +135,17 @@ export class WebsiteBuilder extends Component {
     }
 
     addSystrayItems() {
-        registry
-            .category("systray")
-            .add(
-                "website.WebsiteSystrayItem",
-                { Component: WebsiteSystrayItem, props: this.systrayProps },
-                { sequence: -100 }
-            );
+        if (!registry.category("systray").get("website.WebsiteSystrayItem", false)) {        
```

Comment:
```suggestion
        if (!registry.category("systray").contains("website.WebsiteSystrayItem")) {
```

(also, I see you have a bunch of spaces at the end of the line, you should make sure your linter is working to trim those trailing spaces :) )